apps/site/pages/docs/customizing-pages.mdx

```diff
@@ -0,0 +1,251 @@
+import { Tab, Tabs, Callout } from 'nextra-theme-docs'
+import { useFormattedPrice } from 'site/components/utilities/usePriceFormatter'
+import { product } from 'site/mocks/product'
+import { OverviewSection } from 'site/components/OverviewSection'
+import {
+  Banner,
+  BannerContent,
+  ProductCard,
+  ProductCardContent,
+  ProductCardImage,
+} from '@faststore/ui'
+
+<header className="hero">
+
+# Customizing Pages
+
+We provide a couple mixins to help you styles your page sections.
+
+</header>
+
+## Styling a section
```

Comment:
We probably need to explain what it's a page section here.
I hope we can have this explanation in this [page](https://www.faststore.dev/docs/reference-guides/list-of-available-native-sections) soon.